public/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+	<!-- This is a loader for JXML Applications / Modules -->
+	<head>
+		<script src="lib/system.js"></script>
+	</head>
+	<body>
+		<script>
+			System.config({ baseURL: 'lib' });
+
+			var params = {
+				app: 'jx/example/KitchenSink',
+				renderer: 'jx/ui/html/HTMLRenderer'
+			};
+
+			location.search.replace(/\??(([^=]*)=)?([^&]+)&?/g, function(_, _, k, v) {
```

Comment:
the problem happens when we type `?app=jx/example/KitchenSink/`, the regex
does't account for the last `/` and tries to load `KitchenSink/.jxml`.

On Wed, Jan 7, 2015 at 5:54 PM, Kwok Yang Bin <notifications@github.com>
wrote:

> In public/index.html
> <https://github.com/jxml/jxml/pull/4#discussion-diff-22578255>:
>
> > +<!DOCTYPE html>
> > +<html>
> > +	<!-- This is a loader for JXML Applications / Modules -->
> > +	<head>
> > +		<script src="lib/system.js"></script>
> > +	</head>
> > +	<body>
> > +		<script>
> > +			System.config({ baseURL: 'lib' });
> > +
> > +			var params = {
> > +				app: 'jx/example/KitchenSink',
> > +				renderer: 'jx/ui/html/HTMLRenderer'
> > +			};
> > +
> > +			location.search.replace(/\??(([^=]*)=)?([^&]+)&?/g, function(_, _, k, v) {
>
> Can you give an example?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jxml/jxml/pull/4/files#r22578255>.
>